english/stargazing/stargazing.md

```diff
@@ -0,0 +1,14 @@
+"Look at the stars" she said as she looked at him.
+
+They were laying on the lush green Great Lawn of the Central park.
+
+She likes admiring the sky because she could imagine themselves in another planet, maybe outside of the galaxy.
+
+"They are shining for you" his voice sounded soft. He seemed to be calm, even when he had a lot of panic on the inside. 
+
+"They are scared because they know they'll never see us together again".
+
+He gave her a quick peck on her lips and took her hand to hold it like he never did before. 
+
+They closed their eyes and a couple of minutes later everything vanished into the bright sunlight coming through a window.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.